docs/starters.yml

```diff
@@ -6959,3 +6959,21 @@
     - Uses SCSS for easy-to-understand naming
     - Google Analytics support
     - Responsive design
+- url: https://gatsby-directus-starter.netlify.app/
+  repo: https://github.com/muhzulzidan/gatsby-starter-directus
+  description: A starter template to build websites using  Directus as CMS & Gatsby for FrontEnd 
+  tags:
```

Comment:
Refer to [categories.yml](https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml) to see the valid tags that can be added here. You cannot have tags like `Styling:Sass` (Use instead: `Styling:SCSS`), `CMS:Directus` etc. as they aren't part of `categories.yml` if you believe they are missing you can request additions to the file. However, we recommend using the available categories.

Reply:
so i can't do pull request using github UI ?

Reply:
i made the changes here : https://github.com/gatsbyjs/gatsby/pull/26457

and I think it is still not accepted by circleci...